src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/dependency/DependencyQueueTaskDispatcher.java

```diff
@@ -188,18 +191,50 @@ public CauseOfBlockage canRun(Queue.Item item) {
             return new BecauseDependentBuildIsBuilding(blockingProjects);
         } else {
             logger.info("No active dependencies on project: {} , it will now build", p);
+
+            ToGerritRunListener toGerritRunListener = ToGerritRunListener.getInstance();
+            if (toGerritRunListener != null) {
+                List<ParameterValue> parameters = new ArrayList<ParameterValue>();
+                StringBuilder actualDependencies = new StringBuilder();
+                for (Run run : toGerritRunListener.getRuns(event)) {
+                    if (dependencies.contains(run.getParent())) {
+                        String originalName = run.getParent().getFullName();
+                        String keyName = originalName.replaceAll("[^a-zA-Z0-9]+", "_");
+                        String prefix = "DEP_" + keyName;
+                        String number = String.valueOf(run.getNumber());
+                        String result = run.getResult().toString();
+                        parameters.add(new StringParameterValue(prefix + "_BUILD_NAME", originalName));
```

Comment:
@rsandell, I did it... and only after that I faced the problem. This Action doesn't support Pipeline jobs... https://issues.jenkins-ci.org/browse/JENKINS-29537